eve/main.yml

```diff
@@ -1315,6 +1317,103 @@ stages:
+            -printf '%f\n' | sort | tail -n1"
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Retrieve the backup archive from the bootstrap node
+          command: >
+            scp -F ssh_config
+            bootstrap:"/var/lib/metalk8s/%(prop:bootstrap_backup_archive)s"
+            /tmp
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Destroy the bootstrap node
+          env: *_env_terraform_multiple_nodes
+          command: >
+            terraform destroy -auto-approve
+            -target openstack_compute_instance_v2.bootstrap
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - SetPropertyFromCommand:
+          name: Set node1 etcd container id
+          property: node1_etcd_container_id
+          command: >
+            ssh -F ssh_config node1
+            sudo crictl ps -q --label io.kubernetes.pod.namespace=kube-system
+            --label io.kubernetes.container.name=etcd --state Running
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - SetPropertyFromCommand:
+          name: Set boostrap node etcd member id
+          property: bootstrap_etcd_member_id
+          command: >
+            ssh -F ssh_config node1
+            sudo crictl exec -i "%(prop:node1_etcd_container_id)s" sh -c \"
+            ETCDCTL_API=3 etcdctl --endpoints https://127.0.0.1:2379
+            --cert /etc/kubernetes/pki/etcd/server.crt
+            --key /etc/kubernetes/pki/etcd/server.key
+            --cacert /etc/kubernetes/pki/etcd/ca.crt
+            member list\" | awk -F ', ' '$3 ~ "bootstrap" { print $1 }'
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Remove bootstrap node from etcd members
+          command: >
+            ssh -F ssh_config node1
+            sudo crictl exec -i "%(prop:node1_etcd_container_id)s" sh -c \"
+            ETCDCTL_API=3 etcdctl --endpoints https://127.0.0.1:2379
+            --cert /etc/kubernetes/pki/etcd/server.crt
+            --key /etc/kubernetes/pki/etcd/server.key
+            --cacert /etc/kubernetes/pki/etcd/ca.crt
+            member remove %(prop:bootstrap_etcd_member_id)s\"
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Recreate the bootstrap node
+          env: *_env_terraform_multiple_nodes
+          command:
+            terraform apply -auto-approve -refresh
+            -target openstack_compute_instance_v2.bootstrap
+            -target null_resource.ssh_config
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
```

Comment:
Maybe could be simpler/better to install `etcdctl` on the bastion and directly do the member list and remove from here using the right endpoint